pkg/virt-api/rest/spice.go

```diff
@@ -74,9 +68,12 @@ func spiceFromVM(vm *v1.VirtualMachine) (*v1.Spice, error) {
 				Host: d.Host,
 				Port: d.Port,
 			}
-			if spiceProxy != "" {
-				spice.Info.Proxy = fmt.Sprintf("http://%s", spiceProxy)
+
+			if proxyIP, ok := config.Get("spice-proxy.ip"); ok {
+				proxyPort, _ := config.Get("spice-proxy.port")
+				spice.Info.Proxy = fmt.Sprintf("http://%s:%s", proxyIP, proxyPort)
```

Comment:
I think it would make sense to combine everything to just one setting. So one would would set `http(s)://192.168.200.2:8183`, instead of having to provide two numbers:

```yaml
data:
  spice-proxy: http://192.168.200.2:8183
```

Reply:
Thanks, updated